src/TibiaHousesOverviewV3.go

```diff
@@ -62,7 +62,7 @@ func TibiaHousesOverviewV3Impl(c *gin.Context, world string, town string) Houses
 			Method: resty.MethodGet,
 			URL:    "https://www.tibia.com/community/?subtopic=houses&world=" + TibiadataQueryEscapeStringV3(world) + "&town=" + TibiadataQueryEscapeStringV3(town) + "&type=" + TibiadataQueryEscapeStringV3(HouseType),
 		}
-		BoxContentHTML, err := TibiadataHTMLDataCollectorV3(tibiadataRequest)
+		BoxContentHTML, err := htmlDataCollector(tibiadataRequest)
```

Comment:
this was interesting.. why would we need this separate function instead of the normal one?

Reply:
If you look at the unit tests you'll see why this is needed. It allows us to switch out how the data is grabbed. In normal usage we use HTTP to reach out to Tibia.com, in the unit tests we just read it from disk.